Cpp/odin-views/password.cpp

```diff
@@ -185,5 +186,45 @@ namespace {
         }
     } c_reset_password;
 
+    const class password_hash : public fostlib::urlhandler::view {
+      public:
+        password_hash() : view("odin.password.hash") {}
+
+        std::pair<boost::shared_ptr<fostlib::mime>, int> operator()(
+                const fostlib::json &config,
+                const fostlib::string &path,
+                fostlib::http::server::request &req,
+                const fostlib::host &host) const {
+            if (!config.has_key("hash") || !config.has_key("verify")
```

Comment:
Should probably make sure that the method is `POST` too shouldn't it?

Reply:
I think we should support every method that has a request body (POST, PUT, PATCH)

Reply:
Yeah